web/src/js/components/FlowView/Messages.jsx

```diff
@@ -68,17 +68,18 @@ function ResponseLine({ flow, readonly, updateFlow }) {
 
 const Message = connect(
     state => ({
-        flow: state.ui.flow.modifiedFlow || state.flows.byId[state.flows.selected[0]],
+        flow: state.flows.byId[state.flows.selected[0]],
```

Comment:
i removed modifiedFlow, because otherwise react js propagates the changes down to the contentloader and then the content will be reloaded so i cannot edit anything